app/views/annotator/index.html.haml

```diff
@@ -1,180 +1,120 @@
-                  %span.popup_container
-                    %span.bp_popup_link_container
-                      %a#filter_ontologies.bp_popup_link{:href => "javascript:void(0);"}= t('filter')
-                    %div#ontology_filter_list.bp_popup_list
-                %th{class: "match_type"}
-                  = t('type')
-                  %span.popup_container
-                    %span.bp_popup_link_container
-                      %a#filter_match_type.bp_popup_link{:href => "javascript:void(0);"}= t('filter')
-                    %div#match_type_filter_list.bp_popup_list
-                %th= t('umls_sem_type')
-                %th{class: "match_context"}= t('context')
-                %th
-                  = t('matched_class')
-                  %span.popup_container
-                    %span.bp_popup_link_container
-                      %a#filter_matched_classes.bp_popup_link{:href => "javascript:void(0);"}= t('filter')
-                    %div#matched_classes_filter_list.bp_popup_list
-                %th
-                  = t('matched_ontology')
-                  %span.popup_container
-                    %span.bp_popup_link_container
-                      %a#filter_matched_ontologies.bp_popup_link{:href => "javascript:void(0);"}= t('filter')
-                    %div#matched_ontology_filter_list.bp_popup_list
-                %th= t('score')
-                %th= t('negation')
-                %th= t('experiencer')
-                %th= t('temporality')
-                %th= t('certainty')
-            %tbody
-          %div.my-3
-            %b= t('format_results')
-          %div.my-3
-            %span#download_links_tabdelimited.link_button.ui_button
-            %span#download_links_json.link_button.ui_button
-            %span#download_links_rdf.link_button.ui_button
-            %span#download_links_text.link_button.ui_button
-            -# %span#download_links_xml.link_button.ui_button
-          %div.mt-3
-            = link_to(t('reproduce_results'))
-            %span#annotator_parameters
-          %div.mb-4
-            = t('additional_parameters')
-            = link_to('Annotator API documentation', "#{$REST_URL}/documentation#nav_annotator", target: "_blank")
-            
-:javascript
-  window.addEventListener("load", function() {
-    if(document.getElementById("annotation_text").value != ''){
-      document.getElementById("annotator_button").click()
-      window.scrollBy(0, 1080);
-    }
-  });
\ No newline at end of file
+.annotator-page-container{'data-controller': 'annotator'}
+  .annotator-page-subcontainer
+    .annotator-page-title
+      .text
+        = @page_name
+      .line
+    .annotator-page-decription
+      = t('annotator.description')
+    = form_tag(@form_url, method: :get, 'data-turbo': true, novalidate: true) do
+      .annotator-page-inputs{'data-controller': 'reveal-component'}
+        .inputs
+          %div
+            .annotator-page-text-area{'data-controller': 'sample-text'}
```

Comment:
this need to be extracted as a component or helper as reused in  recommender and homepage